src/coreclr/nativeaot/Runtime.Base/src/System/Runtime/ExceptionHandling.cs

```diff
@@ -74,7 +74,7 @@ private struct EHEnum
 
         // This is a fail-fast function used by the runtime as a last resort that will terminate the process with
         // as little effort as possible. No guarantee is made about the semantics of this fail-fast.
-        internal static void FallbackFailFast(RhFailFastReason reason, object unhandledException)
+        internal static void FallbackFailFast(RhFailFastReason _ /*reason*/, object _1 /*unhandledException*/)
```

Comment:
I'm not sure it looks like some nativeaot incomplete feature. @MichalStrehovsky do you know?

Reply:
Yes, probably the intent was to provide those details. If we have callsites that provide the arguments, it will be easier to add this in the future. I would keep the parameters for the future.

Reply:
ok, updated